src/main/java/com/binance/api/client/domain/event/AllMarketTickersEvent.java

```diff
@@ -0,0 +1,257 @@
+  }
+
+  public double getPreviousDaysClosePrice() {
+    return previousDaysClosePrice;
+  }
+
+  public void setPreviousDaysClosePrice(double previousDaysClosePrice) {
+    this.previousDaysClosePrice = previousDaysClosePrice;
+  }
+
+  public double getCurrentDaysClosePrice() {
+    return currentDaysClosePrice;
+  }
+
+  public void setCurrentDaysClosePrice(double currentDaysClosePrice) {
+    this.currentDaysClosePrice = currentDaysClosePrice;
+  }
+
+  public long getCloseTradesQuantity() {
+    return closeTradesQuantity;
+  }
+
+  public void setCloseTradesQuantity(long closeTradesQuantity) {
+    this.closeTradesQuantity = closeTradesQuantity;
+  }
+
+  public double getBestAskPrice() {
+    return bestAskPrice;
+  }
+
+  public void setBestAskPrice(double bestAskPrice) {
+    this.bestAskPrice = bestAskPrice;
+  }
+
+  public long getBestAskQuantity() {
+    return bestAskQuantity;
+  }
+
+  public void setBestAskQuantity(long bestAskQuantity) {
+    this.bestAskQuantity = bestAskQuantity;
+  }
+
+  public double getOpenPrice() {
+    return openPrice;
+  }
+
+  public void setOpenPrice(double openPrice) {
+    this.openPrice = openPrice;
+  }
+
+  public double getHighPrice() {
+    return highPrice;
+  }
+
+  public void setHighPrice(double highPrice) {
+    this.highPrice = highPrice;
+  }
+
+  public double getLowPrice() {
+    return lowPrice;
+  }
+
+  public void setLowPrice(double lowPrice) {
+    this.lowPrice = lowPrice;
+  }
+
+  public long getTotalTradedBaseAssetVolume() {
+    return totalTradedBaseAssetVolume;
+  }
+
+  public void setTotalTradedBaseAssetVolume(long totalTradedBaseAssetVolume) {
+    this.totalTradedBaseAssetVolume = totalTradedBaseAssetVolume;
+  }
+
+  public long getTotalTradedQuoteAssetVolume() {
+    return totalTradedQuoteAssetVolume;
+  }
+
+  public void setTotalTradedQuoteAssetVolume(long totalTradedQuoteAssetVolume) {
+    this.totalTradedQuoteAssetVolume = totalTradedQuoteAssetVolume;
+  }
+
+  public long getStatisticesOpenTime() {
+    return statisticesOpenTime;
+  }
+
+  public void setStatisticesOpenTime(long statisticesOpenTime) {
+    this.statisticesOpenTime = statisticesOpenTime;
+  }
+
+  public long getStatisticesCloseTime() {
+    return statisticesCloseTime;
+  }
+
+  public void setStatisticesCloseTime(long statisticesCloseTime) {
+    this.statisticesCloseTime = statisticesCloseTime;
+  }
+
+  public long getFirstTradeId() {
+    return firstTradeId;
+  }
+
+  public void setFirstTradeId(long firstTradeId) {
+    this.firstTradeId = firstTradeId;
+  }
+
+  public long getLastTradeId() {
+    return lastTradeId;
+  }
+
+  public void setLastTradeId(long lastTradeId) {
+    this.lastTradeId = lastTradeId;
+  }
+
+  public long getTotalNumberOfTrades() {
+    return totalNumberOfTrades;
+  }
+
+  public void setTotalNumberOfTrades(long totalNumberOfTrades) {
+    this.totalNumberOfTrades = totalNumberOfTrades;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Could you please replace this to use the `ToStringBuilder` that is used in other transfer objects? Thanks.